components/api_request_helper/api_request_helper.cc

```diff
@@ -94,6 +116,25 @@ bool APIRequestResult::IsResponseCodeValid() const {
   return response_code_ >= 100 && response_code_ <= 599;
 }
 
+base::Value APIRequestResult::TakeBody() {
+#if DCHECK_IS_ON()
+  DCHECK(!body_consumed_);
```

Comment:
I wonder if this should just be a `CHECK`? Perf impact will be pretty minor, and taking twice wouldn't be ideal.

Reply:
Ok, this makes sense. Replaced to `CHECK`.